### PR TITLE
rds for staging

### PR DESCRIPTION
**Pull request by JWanderema**

### CODE REVIEW

File: tests/e2e_test.py [hunk 1]
```diff
@@ -108,8 +108,11 @@ def test_ssm_rds_password_string_type(self):
     def test_iam_role(self):
         self.assertEqual(self.result["root_modules"]["aws_iam_role.postgres"]["name"], "rds-postgres-role-internal-tableau-apps-preprod-dq")
 
-    def test_staging_instance_tag(self):
-        self.assertEqual(self.result["root_modules"]["aws_instance.int_tableau_linux_staging"]["tags.Name"], "ec2-staging-internal-tableau-apps-preprod-dq")
+    #def test_staging_instance_tag(self):
```
Comment (david-wirelab) [inline tests/e2e_test.py]: Why is this commented out? If it's no longer required please remove instead.

File: rds.tf [hunk 2]
```diff
@@ -202,9 +202,9 @@ resource "aws_db_instance" "internal_reporting_snapshot_qa" {
   }
 }
 
-resource "aws_db_instance" "internal_reporting_snapshot_prod_test" {
+resource "aws_db_instance" "prod_staging_postgres" {
```
[the remark this answers is not in the document]
Comment (JWanderema) [reply rds.tf]: how about "internal_reporting_snapshot_prod_staging"

[the remark this answers is not in the document]
Comment (david-wirelab) [reply rds.tf]: ok

File: tests/e2e_test.py (hunk 1 above)
Comment (JWanderema) [reply tests/e2e_test.py]: I forgot to undo that. I was running local test in NotProd and didnt want to change the main file. Ill correct that.

**Pull request closed without merging**